src/portal/src/i18n/lang/fr-fr-lang.json

```diff
@@ -128,8 +128,6 @@
         "CLI_PASSWORD_TIP": "Vous pouvez utiliser ce secret comme mot de passe quand vous utilisez la CLI de docker/helm pour accéder à harbor.",
         "COPY_SUCCESS": "Copie effectuée",
         "COPY_ERROR": "Copie échouée",
-        "ADMIN_CIL_SECRET_BUTTON": "GENERER UN SECRET",
-        "ADMIN_CIL_SECRET_RESET_BUTTON": "Téléverser votre propre Secret",
         "NEW_SECRET": "Secret",
```

Comment:
```suggestion
        "NEW_SECRET": "Nouveau Secret",
```

Reply:
In english it's translated as "Secret", too, and in chinese too, so I'll let "secret"